### DISCUSSION TITLE
Sign with TSA

### DISCUSSION THREAD

**Discussion by pibia** [lsnepomuceno/laravel-a1-pdf-sign#97] [Ideas]
is it possible to apply the timestamp (even from free services like freetsa.org)?

**Comment by lsnepomuceno** [comment]
Hello @pibia ,

As I had informed in the issue below, I will continue to include TSA signatures in the package I am developing.

https://github.com/lsnepomuceno/laravel-a1-pdf-sign/issues/45

**Comment by lsnepomuceno** [comment]
See the new v2.0 version: https://github.com/lsnepomuceno/laravel-a1-pdf-sign/releases/tag/2.0.0
